packages/contracts/src/contracts/current/protocol/Exchange/MixinSettlement.sol

```diff
@@ -41,12 +40,20 @@ contract MixinSettlement is
         return ZRX_PROXY_DATA;
     }
 
-    function MixinSettlement(bytes memory _zrxProxyData)
+    /// TODO: _zrxProxyData should be a constant in production.
```

Comment:
When will this TODO be addressed?

Reply:
During our optimization phase - much easier to write tests as is.